### ISSUE TITLE
Markdown title anchor are not working

### ISSUE THREAD

**Issue by MichielDG** [nhn/tui.editor#803]
## Version
v1.4.10

## Development Environment
Chrome 80.0.3987.132

## Current Behavior
Write a description of the current operation. You can add sample code, 'CodePen' or 'jsfiddle' links.

```md
// Write example code
Go to section
* [Hello](#hello)  
* [Hello World](#hello-world)
* [Another section](#new-section)    <-- it's called 'Another section' in this list but refers to 'New section'
```

## Expected Behavior
The example above does not work in the tui editor/viewer. This feature would be handy for documentation purposes.

**Comment by seonim-ryu** [comment]
@MichielDG I'm sorry.. I'm not sure what function you're talking about.

Do you mean that the function you are talking about moves to "new-section" when you click "Another section" when you click in the preview area?

![preview](https://user-images.githubusercontent.com/18183560/77137465-70a77600-6ab1-11ea-8c67-17373eeb9ce3.png)

I would appreciate it if you could explain it in more detail.

**Comment by MichielDG** [comment]
> @MichielDG I'm sorry.. I'm not sure what function you're talking about.
> 
> Do you mean that the function you are talking about moves to "new-section" when you click "Another section" when you click in the preview area?
> 
> ![preview](https://user-images.githubusercontent.com/18183560/77137465-70a77600-6ab1-11ea-8c67-17373eeb9ce3.png)
> 
> I would appreciate it if you could explain it in more detail.

Yes that is exactly what I want. When you click on link Another section it moves to the title new section.

**Comment by seonim-ryu** [comment]
@MichielDG This feature should be considered when improving the preview function of the Markdown editor. I think WYSIWYG should also be considered. It will be difficult to deal with right now. Anyway, thanks for the comments! :)
